src/Symfony/Component/Form/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ CHANGELOG
  * Deprecate calling `Button/Form::setParent()`, `ButtonBuilder/FormConfigBuilder::setDataMapper()`, `TransformationFailedException::setInvalidMessage()` without arguments
  * Change the signature of `FormConfigBuilderInterface::setDataMapper()` to `setDataMapper(?DataMapperInterface)`
  * Change the signature of `FormInterface::setParent()` to `setParent(?self)`
+ * Add a `placeholder_disabled` option to `ChoiceType` (resolves #22318, #38958)
```

Comment:
as far as I know, there is no need of writing `(resolves #22318, #38958)`
the PR desc is the right place (and you already write this ;))

Reply:
Removed.